spec.html

```diff
@@ -34690,7 +34690,7 @@ <h1>Properties of _TypedArray_ Instances</h1>
 
 <!-- es6num="23" -->
 <emu-clause id="sec-keyed-collection">
-  <h1>Keyed Collection</h1>
```

Comment:
This fixes the issue, but breaks any old links pointing to "keyed-collection". Can we find a way to preserve the old link as well?